src/dev-app/chips/chips-demo.html

```diff
@@ -261,6 +261,17 @@ <h4>Single selection without checkmark indicator.</h4>
       </mat-chip-listbox>
 
       The selected color is {{selectedColor}}.
+
+      <h4>Single selection with stacked appearance.</h4>
+
+      <mat-chip-listbox [(ngModel)]="selectedColor" class="mat-mdc-chip-set-stacked">
```

Comment:
Should we have a suffix icon here as well to ensure that the new winder primary action doesn't push it out?

Reply:
Hello, I don't think I understand. I didn't actually make the visual chip look any wider, I just made the `<button/>` element grow to fill the width of the chip.

This PR does not intend to make visual changes. Here's what it looks like.
<img width="440" alt="image" src="https://github.com/angular/components/assets/7720245/597d444d-94b7-407b-ac43-5a31cac51636">

Reply:
Good idea. I agree since it was not trivial for me to correctly write the SASS for this.

I added trailing icons to the demo.